book/src/preface.asciidoc

```diff
@@ -20,7 +20,7 @@ Julia fue lanzado originalmente en 2012 por Alan Edelman, Stefan Karpinski, Jeff
 La elección de un lenguaje de programación es siempre subjetiva. Para mí, las siguientes características de Julia son decisivas:
 
 - Julia está desarrollado como un lenguaje de programación de alto rendimiento.
-- Julia usa envío múltiple “multiple dispatch”, que le permite al programador elegir entre diferentes patrones de programación de acuerdo a la aplicación.
+- Julia usa envío múltiple (“multiple dispatch” en ingles), que le permite al programador elegir entre diferentes patrones de programación de acuerdo a la aplicación.
```

Comment:
inglés instead of ingles (can you run a search and replace over the document to change this?, i think that this maybe is present elsewhere)